core/src/epicli/data/common/validation/configuration/helm.yml

```diff
@@ -1 +1,8 @@
-$ref: '#/definitions/unvalidated_specification'
\ No newline at end of file
+"$id": "#/specification"
+title: "Helm specification schema"
+description: "Helm specification schema"
+type: object
+properties:
+  apache_epirepo_path:
+    type: string
+  
```

Comment:
need to change

Reply:
Not sure what you want me to change, as the default configuration looks like this now:

```yaml
kind: configuration/helm
title: "Helm"
name: default
specification:
  apache_epirepo_path: "/var/www/html/epirepo" 
```

Reply:
Whitespaces and newline, it's not correctly displayed here 

![image](https://user-images.githubusercontent.com/6271862/127451103-cb118bdc-2d2d-4400-bfec-8deb703ac6a4.png)

Reply:
Nevermind, I see now;)